SmartDeviceLink/SDLStreamingMediaLifecycleManager.m

```diff
@@ -103,6 +104,12 @@ - (instancetype)initWithConnectionManager:(id<SDLConnectionManagerType>)connecti
     SDLLogV(@"Creating StreamingLifecycleManager");
     _connectionManager = connectionManager;
 
+    if (configuration.window != nil) {
+        _hapticInterface = [[SDLHapticManager alloc] initWithWindow:configuration.window connectionManager:_connectionManager];
+    }
+    //    __weak typeof(_hapticInterface) weakHaptic = _hapticInterface;
+    _touchManager = [[SDLTouchManager alloc] initWithHitTester:_hapticInterface];
+
```

Comment:
I am getting a warning for `_touchManager` initialization.
> Sending 'id<SDLHapticInterface>  _Nullable __strong' to parameter of incompatible type 'id<SDLHapticHitTester> _Nullable'

Casting `_hapticInterface` to `id` clears this warning for me.
`_touchManager = [[SDLTouchManager alloc] initWithHitTester:(id)_hapticInterface]`